server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -1080,20 +1080,20 @@ export class AgenticChatController implements ChatHandlers {
                 messageId: toolUse.toolUseId,
                 type: 'tool',
                 body: '```shell\n' + (toolUse.input as unknown as ExecuteBashParams).command + '\n',
+                ...(isAccept ? {} : { muted: true }),
                 header: {
                     body: 'shell',
                     ...(isAccept
                         ? {}
                         : {
                               status: {
-                                  status: 'error',
                                   icon: 'cancel',
                                   text: 'Rejected',
                               },
                           }),
                     buttons: isAccept ? [{ id: 'stop-shell-command', text: 'Stop', icon: 'stop' }] : [],
                 },
-            }
+            } as any
```

Comment:
We should not be doing it. `muted` has to be in `ChatResult` explicitly, otherwise there's no interface anymore, we go back to Mynah types.

Reply:
+1, no `any` please. I think we need to do a conditional check in mynahUi instead of passing in a muted value here